resources/views/search.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layouts.app')
+
+@section('title')
+    Dashboard
+@endsection
+
+@section('content')
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                @if($userSearchResponse->count() == 0)
+                    <div class="col-md-8 offset-md-2">
+                        <div class="alert my-3 alert-danger" role="alert">
+                            Keinen Benutzer gefunden!
+                            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                                <span aria-hidden="true">×</span>
+                            </button>
+                        </div>
+                    </div>
+                @endif
+                @foreach($userSearchResponse as $user)
+                <div class="card status mt-3" id="status-1" data-body="" data-date="Mittwoch, 07. Oktober 2020">
```

Comment:
You should check whether the ``data-body`` and ``data-date`` are correct.

Reply:
it's not needed...